spec/requests/requests_spec.rb

```diff
@@ -111,13 +111,13 @@
     end
   end
 
-  # Test suite for PUT /workflows/:workflow_id/requests
+  # Test suite for POST /workflows/:workflow_id/requests
   describe 'POST /workflows/:workflow_id/requests' do
     let(:item) { { 'disk' => '100GB' } }
-    let(:valid_attributes) { { requester: '1234', name: 'Visit Narnia', content: JSON.generate(item), decision: 'unknown', state: 'pending' } }
+    let(:valid_attributes) { { requester: '1234', name: 'Visit Narnia', content: JSON.generate(item) } }
 
     context 'when request attributes are valid' do
-      before { post "/workflows/#{workflow_id}/requests", params: valid_attributes, headers: admin_encode_key }
```

Comment:
Was the header constraint pulled out in an earlier PR?

Reply:
Post a request is user's operation. It shouldn't have admin header constraint. I corrected it here.